.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: node_js
 node_js:
-  - "node"
+  - "8"
```

Comment:
Bit late to the party, but should this reflect the version in serverless.yml? or are we just anticipating the next PR :)